lib/elixir/unicode/unicode.ex

```diff
@@ -308,28 +308,23 @@ defmodule String.Graphemes do
   end
 
   # Handle Hangul L
-  for codepoint <- cluster["L"] do
-    defp next_hangul_l_size(<<unquote(codepoint), rest::binary>>, size) do
-      next_hangul_l_size(rest, size + unquote(byte_size(codepoint)))
-    end
-  end
 
-  for codepoint <- cluster["LV"] do
-    defp next_hangul_l_size(<<unquote(codepoint), rest::binary>>, size) do
-      next_hangul_v_size(rest, size + unquote(byte_size(codepoint)))
-    end
+  @cluster_reverse_map (for {class, list} <- cluster, codepoint <- list do
+    {codepoint, class}
   end
+  |> Map.new)
```

Comment:
`for {class, list} <- cluster, codepoint <- list, into: %{}, do: {codepoint, class}`